### PR TITLE
ITSNP-8 :Design post view components

### LINKED ISSUE #8: Design post view components (m6)
### Requirements
1. The Post Components need to be pixel perfect to the Figma design,
2. Components must be responsive for all screen
3. props should be maintained if needed

### TODO : Components

- [x] 1) Design Blog post header Component, here is a figma link :
https://www.figma.com/file/y9aknqd9ijJnyBF67fS6mF/UI-UX-For-temp-madhyam?node-id=443%3A929

- [x] 2) Design blog post footer Components, here is a figma link : https://www.figma.com/file/y9aknqd9ijJnyBF67fS6mF/UI-UX-For-temp-madhyam?node-id=591%3A195

### PR DESCRIPTION
**Pull request by questbibek**
-Design Blog post header Component was created
-Design blog post footer Components,
-Made responsive also

### CODE REVIEW

**Comment by RudeSoul** [comment]
> -Design Blog post header Component was created
> -Design blog post footer Components,
> -Made responsive also

Not done, must be list. Use spaces between hyphen and the sentence to make them list

**Comment by Nabarajrai** [comment]
done.

File: src/components/Header/design-post/footer/footer.scss [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  justify-content: space-around;
+  width: 100%;
+  height: 70px;
+  align-items: center;
+  .footer_right {
+    .footer_icons {
+      display: flex;
+      gap: 15px;
+    }
+  }
```
Comment (bijudhungel11) [inline src/components/Header/design-post/footer/footer.scss:12]: please use the BEM mixins

File: src/components/Header/design-post/images/comment.svg [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+<path d="M18.7193 21.7001C18.9751 21.591 19.1403 21.3409 19.1403 21.0628V17.5864C19.1403 17.5668 19.1563 17.5509 19.1759 17.5509C20.7332 17.5509 22 16.284 22 14.7268V5.81519C22 5.63721 21.8557 5.49292 21.6777 5.49292C21.4998 5.49292 21.3555 5.63721 21.3555 5.81519V14.7268C21.3555 15.9286 20.3778 16.9063 19.1759 16.9063C18.8009 16.9063 18.4958 17.2114 18.4958 17.5864V21.0628C18.4958 21.106 18.4453 21.1276 18.4141 21.0977L14.376 17.2248C14.1619 17.0195 13.8806 16.9064 13.5839 16.9064H2.82404C1.62224 16.9064 0.644487 15.9287 0.644487 14.7268V12.6526C0.644487 12.4746 0.500198 12.3303 0.322222 12.3303C0.144245 12.3303 -4.19617e-05 12.4746 -4.19617e-05 12.6526V14.7268C-4.19617e-05 16.284 1.2668 17.5509 2.82404 17.5509H13.5839C13.7135 17.5509 13.8364 17.6003 13.9299 17.69L17.9679 21.5628C18.1693 21.7559 18.4644 21.8088 18.7193 21.7001Z" fill="black" fill-opacity="0.6"/>
+</g>
+<defs>
+<clipPath id="clip0">
+<rect width="22" height="22" fill="white" transform="matrix(-1 0 0 1 22 0)"/>
+</clipPath>
+</defs>
+</svg>
```
Comment (bijudhungel11) [inline src/components/Header/design-post/images/comment.svg:15]: please create svg component and export it

File: src/components/Header/design-post/footer/footer.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+import like from "../images/like.svg";
+import comment from "../images/comment.svg";
+import facebook from "../images/facebook.svg";
+import linkendin from "../images/linkedin.svg";
+import more from "../images/more-horizontal.svg";
+import save from "../images/save.svg";
+import twitter from "../images/twitter.svg";
+import "./footer.scss";
+
+const footer = () => {
```
Comment (bijudhungel11) [inline src/components/Header/design-post/footer/footer.js:11]: Component Name should Capital

**Pull request closed without merging**